node/src/components/contract_runtime.rs

```diff
@@ -297,6 +298,43 @@ where
                 }
                 .ignore()
             }
+            Event::Request(ContractRuntimeRequest::IsBonded {
+                state_root_hash,
+                era_id,
+                protocol_version,
+                public_key: validator_key,
+                responder,
+            }) => {
+                trace!("is validator bonded request");
```

Comment:
Maybe log more information:
```suggestion
                trace!(era = %era_id, %public_key, "is validator bonded request");
```

Reply:
Done in [`c0bd13f` (#683)](https://github.com/CasperLabs/casper-node/pull/683/commits/c0bd13f70e41208382667535631dd62c9a381c5c)